charts/eks-anywhere-packages/templates/credentialpackage.yaml

```diff
@@ -26,7 +26,7 @@ spec:
       - matchImages:
         - {{.Values.defaultImageRegistry}}
         profile: "default"
-        secretName: aws-secret
+        secretName: {{ ternary "aws-secret" "aws-config" (empty .Values.awsConfig) }}
```

Comment:
Why are we adding a new secret instead of just modifying aws-secret? What other methods of authn is this adding?

Reply:
Good point. I created a new secret by following this doc https://anywhere.eks.amazonaws.com/docs/packages/credential-provider-package/iam_roles_anywhere/. But after checking it in details, I think it's possible to reuse aws-secret.

> What other methods of authn is this adding?
By making this aws-secret general, the users will be able to use "role-assume", "aws roles anywhere" etc to do authn, reference: https://docs.aws.amazon.com/sdkref/latest/guide/file-format.html.